data/ecosystems/f/flow.toml

```diff
@@ -75,6 +75,7 @@ github_organizations = [
   "https://github.com/FlowFans-Learning",
   "https://github.com/letthemusicpay",
   "https://github.com/LiquidityTokenLabs",
+  "https://github.com/orgs/IncrementFi",
```

Comment:
can you replace this org URL with adding IncrementFi as a subecosystem of Flow (see list at the beginning of this same file)?